packages/migration-graph/src/resolver.ts

```diff
@@ -90,7 +90,13 @@ export class Resolver {
 
     this.graph.addFileToPackage(packageNode, importerAbsPath);
 
+    // We will not attemptt to parse an .hbs
+    if (extname(importerAbsPath) === '.hbs') {
```

Comment:
this makes sense since we need to include the .hbs file in the graph; otherwise the file wont exist for the glint plugin(s). an issue with this is the `rehearsal graph` command will then include this file in the output artifact and log. this is confusing since the .hbs file(s) in question dont have any imports.